<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateSalesReport</class>
 <widget class="QWidget" name="GenerateSalesReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenerateSalesReport</string>
  </property>
  <widget class="QPushButton" name="SalesReportNext">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;To:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Sales Report&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="SalesReportFrom">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;From:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="SalesReportTo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Backbuttom">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>650</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>461</width>
     <height>431</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Item</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Sales</string>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SalePrice</string>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Backbuttom</sender>
   <signal>clicked()</signal>
   <receiver>GenerateSalesReport</receiver>
   <slot>Backbutton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>751</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SalesReportNext</sender>
   <signal>clicked()</signal>
   <receiver>GenerateSalesReport</receiver>
   <slot>Submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Backbutton()</slot>
  <slot>Submit()</slot>
 </slots>
</ui>
